src/main.ts

```diff
@@ -96,7 +96,7 @@ export function validateObjectTree(iobObjects: ObjectWithValue[]): void {
 				if (iobObjects.filter(item => item.id === `${iobIDBasePath}${iobIDName}`).length > 1){
 					throw `Duplicated object ${iobIDBasePath}${iobIDName} defined`
 				}else{
-					if (iobObjects.filter(item => item.id === `${iobIDBasePath}${iobIDName}`).length === 0){
+					if (iobObjects.filter(item => item.id === `${iobIDBasePath}${iobIDName}`).length === 2){
```

Comment:
Are you sure this is correct? Why must there be two objects with that ID?

Reply:
That was totally wrong. Fixed now, see new commit.